### DISCUSSION TITLE
Tracking of balance (unplayable mus) & level of balance to be reached (win%)

### DISCUSSION THREAD

**Discussion by ReLaX82** [TheSuperHackers/GeneralsGamePatch#687] [General]
Its not a change/bug discussion, but its rather a discussion at which state of balance or lets say degree of balance we say its good enough.

Having a chance of winning of 50% is ideal, but not doable in all matchups.

So, what is the goal numer, 30% at least or minimum 35% or even more like 40%?
Always though be able to win at least each 3rd game would be fine for ZH, which would match with 35%.

Also we need to think about how we track unplayable matchups.

Maybe a quite good approach I did in which Domi's and other inputs is used to show how bad balanced 1.04 is.
My approach is that 35% win% to 65% win is acceptable and counted as an playable matchup.
It highlights bad MUs and provides numbers and also patch intermediate results as a graph if we wanted to.

Please see file attached.

**Comment by ReLaX82** [comment]
https://www.gamereplays.org/community/index.php?act=Attach&type=post&id=873962

**Comment by xezon** [comment]
Airforce VS SWG is unplayable matchup. Let's start addressing from there.

**Reply by Madrageeeeee** [reply]
Thinking about it a solution might be that airforce chinooks get a specific weakness for hits from laserlocking (i.e. do more damage than regular hits, lets say 130% compared to current). This would make a well executed 3/4 md drop more viable and successfull in general. An air chinook surviving or dieing makes a big difference. 

This wudnt change gla/china vs air, it wouldnt change the PDL functionality  but would boost usa/swg/laser chances against airforce. 

 Although this deviates from the prices / build times changes approach.

**Reply by xezon** [reply]
I added this proposal as no 12 to
* #638

**Comment by ReLaX82** [comment]
See excel its Set to 95%. 
95% being the highest number and only 5% allowed.

**Comment by MTKing4** [comment]
I think we should try to achieve 40% at least

**Comment by ReLaX82** [comment]
So, 40% of chance SW winning vs Air you mean?
That would be unrealistical which of course depends on the approach you do.

What is an unplayable mus for you guys?

Main idea behind it could also be able to play full Random-only tournaments which not end up in unbalanced mus and potentially a very bad coin flip towards one player.

**Comment by MTKing4** [comment]
I don't think 35-65% winrate is enough if you want to have a full random-only tournaments and be a fair format. For all MUs

**Comment by ReLaX82** [comment]
If I say such number I have always in mind that we start with some at 95%, having 12 faction AND having the aim to not change the feeling of the game.

I understand your thinking, but looking at the improovements 5% win chance and then having 35% is massive.

**Comment by ExiLe-ZH** [comment]
> I don't think 35-65% winrate is enough if you want to have a full random-only tournaments and be a fair format. For all MUs

You can not realistically aim higher with the rock-paper-scissors 12-army balance system zh is designed upon.

**Comment by MTKing4** [comment]
If you set your goals low, chances are you're gonna score low, dream big guys, maybe we will achieve that then!

**Comment by ExiLe-ZH** [comment]
> If you set your goals low, chances are you're gonna score low, dream big guys, maybe we will achieve that then!

Better to set #SMART goals

**Comment by ReLaX82** [comment]
Anyone looked closer in the table and find it a good approach to get an overview no matter what you think its playble in %?

**Comment by MTKing4** [comment]
> > If you set your goals low, chances are you're gonna score low, dream big guys, maybe we will achieve that then!
> 
> Better to set #SMART goals

Having high goals doesn't dictate them being dumb
It just means you'll try harder to fix the problem rather than being lazy about it and rely on untested theories

**Reply by ExiLe-ZH** [reply]
> > > If you set your goals low, chances are you're gonna score low, dream big guys, maybe we will achieve that then!
> > 
> > 
> > Better to set #SMART goals
> 
> Having high goals doesn't dictate them being dumb It just means you'll try harder to fix the problem rather than being lazy about it and rely on untested theories

Not saying it's dumb, saying it's not realistic, which is the R in smart (look up smart goals).

I just don't see how you can make sw/laser/usa that much better vs Air and achieve an average 40% winrate without affecting other matchups or heavily nerfing air.

*anyways, better to talk about actual solution ideas.

**Reply by xezon** [reply]
We will continue our current path. We identify issues and problems and then try to find a solutions to fix or mitigate them. Then we may end up with a product that plays just a bit better than the original without these problems.

**Reply by MTKing4** [reply]
> I just don't see how you can make sw/laser/usa that much better vs Air and achieve an average 40% winrate without affecting other matchups or heavily nerfing air.

I think Air is Overpowered compared to all factions, so all MUs will be slightly changed obviously, i think starting with chinook PDL nerf is a good place to start with. I bet the percentage will change a lot just by that.

**Comment by ReLaX82** [comment]
Thats also okay from my side.
Working towards a goal and dont reach it could be frustrating.

**Comment by RisingZH** [comment]
https://www.gamereplays.org/community/index.php?showtopic=1063583
